handy.h

```diff
@@ -2861,7 +2861,7 @@ last-inclusive range.
                                message ", and will disappear in Perl " when)
 #  define deprecate_fatal_in(when,message) \
               Perl_ck_warner_d(aTHX_ packWARN(WARN_DEPRECATED),    \
-                               message ". Its use will be fatal in Perl " when)
+                               message ", and will become fatal in Perl " when)
```

Comment:
Shouldn't this (and `deprecate_disappears_in`) include the " is deprecated" part of the message too?

Reply:
Ahyes; that's fair. Adjusted and repushed.